src/components/BrowserExamplesMenu/snippets/waitForFunction.js

```diff
@@ -34,7 +34,7 @@ export default async function () {
       polling: 'mutation',
       timeout: 2000,
     });
-    check(ok, { 'waitForFunction successfully resolved': ok.innerHTML() == 'Hello' });
+    await check(ok, { 'waitForFunction successfully resolved': ok.innerHTML() == 'Hello' });
   } finally {
     await page.close();
   }
```

Comment:
Total side note but I hate [this waitForFunction example](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/page/waitforfunction/) 😂 @ankur22 @inancgumus do we have a better example than this? It feels like it is giving mixed messages how to do page locators.

I like the [Playwright equivalent example](https://playwright.dev/docs/api/class-page#page-wait-for-function).